tox.ini

```diff
@@ -10,10 +10,6 @@ deps =
 commands =
     pytest --cov=shub --cov-report=term-missing --cov-report=html --cov-report=xml {posargs:shub tests}
 
-[testenv:min]
-deps =
-    {[testenv]deps}
```

Comment:
Oh, I see this was the same as `py`.

Ideally we should have a `min` env with pinned minimum versions of dependencies, but since it was not there, I’m OK with removing it and adding it back properly later in a separate PR.